src/main/java/core/basesyntax/service/impl/FruitStorageServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operations;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FruitStorageService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStorageServiceImpl implements FruitStorageService {
+    private static final String SPLITTER = ",";
+    private static final int OPERATION = 0;
+    private static final int FRUIT = 1;
+    private static final int AMOUNT = 2;
+    private static final int FIELDS_QUANTITY = 3;
+    private final OperationStrategy operationStrategy;
+    private final ValidateOperationImpl validation = new ValidateOperationImpl();
+    private final FileReader file = new ReadFromCsvFileImpl();
```

Comment:
are our services independent?
can we not use FileReader in FruitStorageServiceImpl?